### PR TITLE
[prometheus-pushgateway] Adds ability to specify metricRelabelings and relabelings

### PR DESCRIPTION
**Pull request by Reuuke**
#### What this PR does / why we need it:

Adds the ability to add `metricRelabelings` and `relabelings` to ServiceMonitor

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-pushgateway/Chart.yaml
Comment (monotek) [inline charts/prometheus-pushgateway/Chart.yaml:5]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-pushgateway/Chart.yaml
Comment (Reuuke) [reply charts/prometheus-pushgateway/Chart.yaml]: Bumped to 1.6.0

**Review by monotek — CHANGES_REQUESTED** [review]
Please sign your commits to get the DCO check passing.

**Comment by monotek** [comment]
@Reuuke 
Please sign your commits to get the DCO check passing.

**Comment by Reuuke** [comment]
> @Reuuke
> Please sign your commits to get the DCO check passing.

Fixed.

**Review by monotek — APPROVED** [review]

**Comment by monotek** [comment]
@torstenwalter 
could you please merge?
waiting for maintainer review is over 2 weeks.

**Review by torstenwalter — APPROVED** [review]

**Pull request merged**
